### ISSUE TITLE
GLTFSCN: Seeing `[SceneKit] Error: SCNSkinner: there must be the same number of bone weights and bone indices` on some models

### ISSUE THREAD

**Issue by Utsira** [warrenm/GLTFKit#11]
When loading certain models into GLTFSCN I get the error `[SceneKit] Error: SCNSkinner: there must be the same number of bone weights and bone indices`. The model loads without a skinner, and is not animatable.

I am on the current master branch of GLTFKit `df82`

The model passes the Kronos GLTF Validator, and the model loads and the animation plays fine in Babylon.js, and in GLTFMTL (I checked it with the GLTFViewer Mac).

I've zipped an example model here: 
[castle-guard-gltf.zip](https://github.com/warrenm/GLTFKit/files/1998968/castle-guard-gltf.zip)

The models in question all came from Adobe Mixamo, via the Blender [Kupoman GLTF exporter](https://github.com/Kupoman/blendergltf). I'm using the Kupoman exporter because it supports multiple animations in one GLTF file. This workflow is described in this tutorial: https://www.donmccurdy.com/2017/11/06/creating-animated-gltf-characters-with-mixamo-and-blender/

**Comment by Utsira** [comment]
The weird thing is, if I log `submesh.accessorsForAttributes[GLTFAttributeSemanticWeights0]` and `submesh.accessorsForAttributes[GLTFAttributeSemanticJoints0]`, they look identical, the `count` is the same. So I'm a bit stumped as to why SceneKit reckons there's a mismatch.

**Comment by warrenm** [comment]
Yeah, I'm looking at this now as well and just noticed the same thing. Strange that it only fails on the Mixamo+Kupoman models...

**Comment by warrenm** [comment]
Can Mixamo export to COLLADA (or something that can be converted to COLLADA)? I'm wondering if we can get this to reproduce in SceneKit without glTF involved...

**Comment by Utsira** [comment]
Yes, Mixamo does have DAE export. I should have time later today to try that

The other thing to try I suppose, would be the Khronos Blender GLTF exporter
